app/src/test/java/org/mozilla/tv/firefox/ext/ContextTest.kt

```diff
@@ -20,35 +20,35 @@ class ContextTest {
 
     @Before
     fun startUp() {
-        context = RuntimeEnvironment.application
+        context = ApplicationProvider.getApplicationContext()
         accessibilityManager = context!!.getAccessibilityManager()
     }
 
     @Test
     fun `isVoiceViewEnabled() returns true when TouchExploration enabled`() {
-        shadowOf(accessibilityManager).isTouchExplorationEnabled = true
+        shadowOf(accessibilityManager).setTouchExplorationEnabled(true)
```

Comment:
Nit: Was this an automated change?  We should prefer property access syntax here.

Reply:
I think this is an API requirement, from when I tried this earlier, because I don't think `AccessibilityManager` actually has a property for this for some reason, maybe the getter/setter isn't a simple value assign.